### DISCUSSION TITLE
So impressive

### DISCUSSION THREAD

**Discussion by watergirl2** [keiwando/toonsquid#558] [General]
Hello

I have been using procreate for a while. Last year I switched to Adobe Fresco because I needed vector brushes. I was hoping procreate dreams will support vector and more. I was super disappointed with Dreams and then I discovered you. 

Wow. What an amazing app! It has everything I would want and more. 

My only question is if there is a way to export / import my Fresco projects with layers to fully move over my new app 

Can't wait to work on my first project and learn more.

Thank you for such an amazing app.

**Comment by Cat-Butler** [comment]
Does Fresco allow you to export each layer as an image sequence? Possibly by hiding all other layers?
That is one possible way to transfer animated layers across.
Individual animation elements might copy and paste (only as pixel content, I think).

Otherwise, I'm not sure there is any close compatibility.

**Comment by Poobio** [comment]
I have been using Rough Animator for years and singing its praises, but like you, also got Procreate Dreams, thinking that, as it was from Procreate, it would be amazing. But no, its like a pre-alpha, missing so many essential basics. Then heard about Toonsquid, and gave it a go. It is what I hoped PDreams would be. The only thing I would say is that RoughAnimator has a slight edge on is the control of keyframes and layers. Especially linked layers, where you can change the timings on one layer and it will affect all of them. Or you can duplicate a layer with blank frames maintaining the timing. Maybe this is possible and I just haven't figured it out yet. Either way, I can't see myself using any other animation app but Toonsquid for the foreseeable future.
